data/campaigns/Heir_To_The_Throne/achievements.cfg

```diff
@@ -1 +1,552 @@
+        icon="units/elves-wood/captain-defend.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/elves-wood/captain-defend.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S2_Guardian_of_the_Blackwater"
+        name=_"Guardian of the Blackwater"
+        name_completed=_"Guardian of the Blackwater"
+        description=_"Defeat the enemy leader in <i>Blackwater Port</i>"
+        description_completed=_"Defeated the enemy leader in <i>Blackwater Port</i>."
+        icon="units/human-loyalists/horseman/horseman.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/human-loyalists/horseman/horseman.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S2_Reward_of_Valor"
+        name=_"Reward of Valor"
+        description=_"Defeated the enemy leader in <i>Blackwater Port</i> on Challenging."
+        icon="units/human-loyalists/knight/knight.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        hidden=yes
+    [/achievement]
+
+    [achievement]
+        id="HttT_S3_Mage_from_the_Hut"
+        name=_"Mage from the Hut"
+        name_completed=_"Mage from the Hut"
+        description=_"Find Elrian the Mage."
+        description_completed=_"Found Elrian the Mage"
+        icon="units/human-magi/mage.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/human-magi/mage.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S4_End_of_Slavery"
+        name=_"End of Slavery"
+        name_completed=_"End of Slavery"
+        description=_"Defeat both enemy leaders in <i>Bay of Pearls</i>"
+        description_completed=_"Defeated both enemy leaders in <i>Bay of Pearls</i>"
+        icon="units/merfolk/fighter.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/merfolk/fighter.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S4_Freedom"
+        name=_"Freedom!"
+        name_completed=_"Freedom!"
+        description=_"Free all the merfolk in <i>Bay of Pearls</i>"
+        description_completed=_"Freed all the merfolk in <i>Bay of Pearls</i>"
+        icon="units/merfolk/fighter.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/merfolk/fighter.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S4_Bugg_Exterminator"
+        name=_"Bugg Exterminator"
```

Comment:
Add a po comment "not a spelling mistake, Bugg is the named Orcish Archer in scenario 4".